_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
+{: .challenge}
+
+- [GeoTIFF](https://www.geospatialworld.net/article/geotiff-a-standard-image-file-format-for-gis-applications/):
+  GeoTIFF is a standard image file format to describe and store raster image data with geographic information.
+  So it can be used by  Geographic Information System (GIS) applications.
+
+  - GeoTIFF format is suitable for a **wide range of applications worldwide**.
+  For example, satellite imaging systems, scanned aerial photography, scanned maps,
+  digital elevation models, or as a result of geographic analyses.
+
+  - It is **independent** of a computer’s architecture, operating system, and graphics hardware.
+
+  - GeoTIFF format has a three-level hierarchy: file header, image file directories, and **data**.
+
+  - GeoTIFF format stores a **broad range of georeferencing information**, catering to geographic
+  as well as projected coordinate systems.
+
+> ## Why GeoTIFF format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the GeoTIFF format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the GeoTIFF format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that GeoTIFF format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> GeoTIFF format is independent of computer’s architecture, operating system, and graphics hardware.
+>> Also, the GeoTIFF format can contain both data and geographic information.
+>>
+>> 2. Therefore, yes, the GeoTIFF format helps to make the data FAIR.
+> {: .solution}
+{: .challenge}
 
- Common file format (FAIR or unFAIR??)
+- [HDF5](https://support.hdfgroup.org/HDF5/doc1.6/UG/03_Model.html):
+The Hierarchical Data Format Version 5, (HDF5) implements a model for managing and storing data,
+developed by the National Center for Supercomputing Applications (NCSA):
+![HDF5]({{ page.root }}/fig/hdf5_data_model.png)
 
-netcdf_data_model
+  - HDF5 is a **general-purpose**, **machine-independent** standard for storing scientific data in files.
 
-GRIB Format
+  - HDF5 files are organized in a hierarchical structure, with two primary structures: groups and datasets.
 
-GeoTIFF
+  - HDF5 dataset: a multidimensional array of **data** elements, together with supporting **metadata**.
 
-AAIGRID
+  - There are four essential classes of metadata: name, datatype, dataspace, and storage layout.
 
-HDF5_data_model
+  - An HDF5 structure is **self-describing**, allowing an application to interpret
+  the structure and contents of a file without any outside information.
 
-netcdf-zarr-data-model
+> ## Why HDF5 format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the HDF5 format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the HDF5 format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that HDF5 format is the standard in your research community and
```

Comment:
Do you have a concrete idea? For I'm not sure I know any (climate-related) community that embraces HDF5

Reply:
I added an example of using the HDF5 format to the text. I hope that it is concrete enough.